src/ecrc-frontend/src/components/page/informationreview/InformationReview.js

```diff
@@ -2,7 +2,7 @@
 import React, { useState, useEffect } from "react";
 import { Redirect, useHistory } from "react-router-dom";
 import PropTypes from "prop-types";
-import axios from "axios";
+// import axios from "axios";
```

Comment:
comment can be removed?

Reply:
Leaving in because this is just for Prod, once this has been pushed to prod, they will be put back in so we can continue testing share so we can get it into prod eventually